### ISSUE TITLE
Glob negation doesn't work in config.input

### ISSUE THREAD

**Issue by Vovan-VE** [i18next/i18next-parser#201]
Hello!

i18next-parser.config.js:
```
module.exports = {
  input: [
    'public/index.html',
    'src/**/*.{ts,tsx}',
    '!src/**/*.test.{ts,tsx}',  // <-- negation doesn't work
  ],
  ...
};
```
Due to following resolving:
https://github.com/i18next/i18next-parser/blob/bb1d82074591ab1cc2ce885d4ea6be26ba0f36d1/bin/cli.js#L66-L68
the negation in `config.input` doesn't work, since the lines above produces glob like
`.../!src/`.

```
[/path/to/project] $ npm run i18n:extract

> app@0.0.0 i18n:extract /path/to/project
> i18next


  i18next Parser
  --------------
  Input:  
  Output: public/locales/$LOCALE/$NAMESPACE.json

  [read]  /path/to/project/!src/foo/bar/fail.test.ts
  [read]  /path/to/project/public/index.html
  [read]  /path/to/project/src/...
```

**Comment by sztahanova** [comment]
Why is this issue closed? Is it fixed? Does it have a workaround? I am running into the same issue and couldn't find any solution...

**Comment by Vovan-VE** [comment]
Was fixed and closed by #203.
